targets/lenovo-x1/guivmExtraModules.nix

```diff
@@ -187,6 +187,10 @@
           # Connect sound device to hosts pulseaudio socket
           "-audiodev"
           "pa,id=pa1,server=unix:/run/pulse/native"
+          "-drive"
+          "file=${pkgs.OVMF.fd}/FV/OVMF_CODE.fd,if=pflash,unit=0,readonly=true"
+          "-drive"
+          "file=${pkgs.OVMF.fd}/FV/OVMF_VARS.fd,if=pflash,unit=1,readonly=true"
```

Comment:
Do you also want to write efi vars? Than this file needs to be writeable. Check this: https://github.com/nix-community/disko/blob/c1cfbfad7cb45f0c177b35b59ba67d1b5fc7ca82/lib/make-disk-image.nix#L38

Reply:
Ah, you set it to readonly...